src/main/java/org/kohsuke/github/ServiceDownException.java

```diff
@@ -0,0 +1,17 @@
+package org.kohsuke.github;
+
+import java.io.IOException;
+
+/**
+ * Special {@link IOException} case for http exceptions, when {@link HttpException} is thrown due to GitHub service
+ * being down.
+ *
+ * @implNote Inherits from {@link HttpException} to not break compatibility with existing code base.
+ *
+ * @author <a href="mailto:cleclerc@cloudbees.com">Cyrille Le Clerc</a>
```

Comment:
I assume you are not Cyrille. 
 
```suggestion
 *
 * @author <a href="mailto:cleclerc@cloudbees.com">Cyrille Le Clerc</a>
```